lib/chef/node/attribute.rb

```diff
@@ -452,9 +452,7 @@ def unlink!(level, *path)
        #
 
       def merged_attributes(*path)
-        # immutablize(
-        merge_all(path)
-        # )
+        immutablize(merge_all(path))
```

Comment:
👍  seems i commented out some code in debugging and forgot to uncomment?  the fact that nothing blew up in tests around this is mildly disturbing.